src/meson.build

```diff
@@ -1,5 +1,27 @@
 a4deps = [fmt_dep, png_dep, jpeg_dep, lcms_dep, tiff_dep, zlib_dep, freetype_dep]
 
+# check whether building against LCMS requires -DCMS_DLL
+a4_extra_cxxargs = []
+if host_machine.system() == 'windows'
+  lcms_test = '''
+#include <lcms2.h>
+
+int main (int argc, char *argv[])
+{
+  cmsGetEncodedCMMversion ();
+  return 0;
+}
+'''
+  cc = meson.get_compiler('c')
+  if not cc.links(
+    lcms_test,
+    dependencies: [lcms_dep],
```

Comment:
This does not work if lcms comes from wrapdb. It is built during regular build so is not available during configuration.

The line below is indented with a tab. That is prohibited but for some reason Github has not run the CI action for this pull request.